vortexasdk/operations.py

```diff
@@ -67,9 +68,13 @@ def __init__(self, resource):
         """
         self._resource = resource
 
-    def search(
-        self, exact_term_match: bool = None, response_type: str = None, headers: dict = None, **api_params
-    ) -> List[dict]:
+    def search_with_client(
```

Comment:
Can you please add a code comment explaining why we need this?

Reply:
In each dataset there is a `search` method that all of the endpoint parameters are passed to by users. This search method then combines all the parameters and passes them to the `search` method inherited from a parent class `Search`. These methods have the same method but different signatures. There's no need for them to have the same name and signature, so I've fixed this typing issue by just renaming the parent `search` method to avoid a signature collision.

An alternative would be to annotate the search methods such that they can accept any kwargs. This `search_with_client` method is not used by consumers so this is not a breaking change.

Reply:
I've added a code comment too